pkg/appmeshinject/jaeger.go

```diff
@@ -0,0 +1,76 @@
+package appmeshinject
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const jaegerTemplate = `
+tracing:
+ http:
+  name: envoy.zipkin
+  typed_config:
+   "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
+   collector_cluster: jaeger
+   collector_endpoint: "/api/v1/spans"
+   shared_span_context: false
+static_resources:
+  clusters:
+  - name: jaeger
+    connect_timeout: 1s
+    type: strict_dns
+    lb_policy: round_robin
+    load_assignment:
+      cluster_name: jaeger
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+           address:
+            socket_address:
+             address: {{ .JaegerAddress }}
+             port_value: {{ .JaegerPort }}
+`
+
+const injectJaegerTemplate = `
+{
+  "command": [
+    "sh",
+    "-c",
+    "cat <<EOF >> /tmp/envoy/envoyconf.yaml{{ .Config }}EOF\n\ncat /tmp/envoy/envoyconf.yaml\n"
```

Comment:
did this jaeger really depends on full content of this Config?
what will //tmp/envoy/envoyconf.yaml do

Reply:
This is the escaped Jaeger template that gets injected to the envoy config file - the term Config caused the confusion, I will rename it in the next commit.